packages/account/src/Components/file-uploader-container/file-uploader.jsx

```diff
@@ -112,6 +112,7 @@ const FileUploader = React.forwardRef(({ onFileDrop, getSocket }, ref) => {
                         })}
                         onClick={removeFile}
                         color='secondary'
+                        data_testid='removeFileIcon'
```

Comment:
```suggestion
                        data_testid='remove_file_icon'
```

also not sure if it should be `data_testid` or `data-testid`

Reply:
Here data_testid is passed as a prop to Icon component and inside the component we have the code to  assign it to data-testid for svg- packages/components/src/components/icon/icon.jsx